introduction_to_amazon_algorithms/jumpstart-foundation-models/text-generation-llama-2-chat.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>This seems to belong in the technical section. How about just highlighting the fact that by using this model, you accept the license </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>